ledger-core-bitcoin/.clang-tidy

```diff
@@ -0,0 +1 @@
+../.clang-tidy
```

Comment:
Do we have to add the symlink here ? A root `.clang-tidy` should be enough isn't it ?

Reply:
The symlink is mandatory because of the current source directory is `ledger-core-bitcoin` so - in a `clang-tidy` point of view - there is no configuration file. 
BTW, you question opens my mind a little bit more and `clang-tidy` has an option to add a configuration path. I will use that instead of the symlink